### PR TITLE
Generate random string fix length

### PR DESCRIPTION
**Pull request by BEAdi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3284)

## Description
Added a check that the length is not more than 10,000, and if so- it is changed.
Updated related YML.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/script-GenerateRandomString.yml [hunk 1]
```diff
@@ -46,6 +46,9 @@ script: >
 
   characters = ""
 
+  if length > 10000:
+      length = 10000
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/script-GenerateRandomString.yml:50]: You can also use the `max` function.
Please make sure to extract the 10000 magic number to a constant.

File: Packs/CommonScripts/Scripts/script-GenerateRandomString.yml [hunk 2]
```diff
@@ -87,35 +90,35 @@ enabled: true
 args:
 - name: Length
   required: true
-  description: Length of the string
+  description: Length of the string. Maximum is 10,000.
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/script-GenerateRandomString.yml:93]: Please make sure to update the readme with these changes.

File: Packs/CommonScripts/Scripts/script-GenerateRandomString.yml (hunk 1 above)
Comment (BEAdi) [reply Packs/CommonScripts/Scripts/script-GenerateRandomString.yml]: Done, I used `min` as we want the lower of the two.

File: Packs/CommonScripts/ReleaseNotes/1_7_7.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### GenerateRandomString
+- Improved implementation with limiting the maximum length of the generated string to be 10,000.
```
Comment (DeanArbel) [inline Packs/CommonScripts/ReleaseNotes/1_7_7.md:4]:
```suggestion
- The max supported *Length* is now up to 10,000 characters. If *Length* of more than 10,000 is provided, it'll be set to 10,000.
```

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Per our conversation, please update the coding standards here, and replace to py3.

File: Packs/CommonScripts/ReleaseNotes/1_7_7.md (hunk 3 above)
Comment (BEAdi) [reply Packs/CommonScripts/ReleaseNotes/1_7_7.md]: Done, changed the first sentence you suggested to `You can now use *Length* only up to 10,000 characters.` to match our available prefixes.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**